presto-verifier/src/main/java/com/facebook/presto/verifier/framework/AbstractVerification.java

```diff
@@ -301,12 +319,14 @@ private VerificationResult concludeVerification(
                         Optional.of(determinismAnalysisDetails) :
                         Optional.empty(),
                 partialResult.getResolveMessage(),
-                buildQueryInfo(
+                skipControl
+                        ? Optional.empty()
+                        : Optional.of(buildQueryInfo(
                         sourceQuery.getControlConfiguration(),
```

Comment:
nit: the indentation is not easy to read now...One idea is 
```java
: Optional.of(
      buildQueryInfo(
            sourceQuery.getControlConfiguration(),
............
```

But that's just my personal way. Feel free to keep the current approach if you see fit.

Reply:
Unfortunately IntelliJ auto-formats to the following if buildQueryInfo is on the next line:
```
skipControl
                        ? Optional.empty()
                        : Optional.of(
                        buildQueryInfo(
                                sourceQuery.getControlConfiguration(),
                                sourceQuery.getControlQuery(),
                                controlChecksumQueryContext,
                                control,
                                controlQueryContext)),
```

Reply:
Moving the ternary operator to the end of the line and indentation works better.